features/logging/cluster_logging_operator.feature

```diff
@@ -152,3 +152,81 @@ Feature: cluster-logging-operator related test
       | card_name | #{cb.card} |
     Then the step should succeed
     """
+
+  # @author gkarager@redhat.com
+  # OCPQE-2773
```

Comment:
@gkarager I have some suggestions:
1. create a TC in Polarion, then add `@case_id` to the scenario 
2. create a new feature file, and put this scenario into the new file
3. when deploying EO/CLO from web console, no need to create namespace before subscribing operator